### DISCUSSION TITLE
Renaming master to main

### DISCUSSION THREAD

**Discussion by haz** [QuMuLab/bauhaus#86] [General]
There any downstream issues that might crop up because of this? Readthedocs, pypi, etc.

Thoughts @KarishmaDaga ?

**Comment by karishmadaga** [comment]
oops just saw this!

I just made 'main' the default branch in readthedocs, so our new PRs are showing there.
I think pypi should be an easy switch, but I gotta look into that.

thanks so much for making this change!! 💛

**Reply by haz** [reply]
I was curious about the readthedocs and pypi integration...had it on my list for this weekend to take a look at ;). Thanks for swapping things out on that front!
